content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/_cq_design_dialog/.content.xml

```diff
@@ -87,6 +87,15 @@
                                     jcr:primaryType="nt:unstructured"
                                     cmp-carousel-v1-dialog-hook="autoplayGroup"/>
                             </autoplayGroup>
+                            <prependControls
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                fieldDescription="When checked, the control elements will be places in front of the carousel items to improve accessibility."
```

Comment:
typo: places -> placed

Reply:
adressed in a407f8573ac5905bd5d9573fdc05fca0a2753c36